.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: objective-c
 os: osx
-osx_image: xcode12.5
+osx_image: xcode13.4
```

Comment:
This change is needed because Xcode 12.5 Swift package manager dependency resolution is not compatible with Xcode 13

Reply:
It's good to keep it fresh anyway. I don't think you can even submit apps to the App Store in Xcode 12 anymore.